### DISCUSSION TITLE
Wireguard on Masters and Sensors

### DISCUSSION THREAD

**Discussion by jrn77478** [Security-Onion-Solutions/securityonion#2130] [Unsupported Versions] [answered]
Running 2.3.10
I'd like to use Wireguard to carry traffic between remote sensors and my SO master. No matter what I've tried, the firewalld configuration will not recognize my Wireguard vpn interface (wg0). Has anyone else worked through this issue?

**Comment by weslambert** [comment, marked as answer]
We can investigate this as part larger compatibility with VPN, here: https://github.com/Security-Onion-Solutions/securityonion/issues/1139
